services/drupal/web/modules/custom/epa_workflow/epa_workflow.install

```diff
@@ -58,3 +58,202 @@ function epa_workflow_uninstall() {
+  $formatted_three_weeks = $three_weeks->format('Y-m-d\TH:i:s');
+
+  if (!isset($sandbox['total'])) {
+    // Query all published nodes that don't have a scheduled transition date, but
+    // do have a review deadline
+    $result = \Drupal::entityQuery('node')
+      ->condition('status', 1)
+      ->condition('field_review_deadline', $formatted_three_weeks, '<=')
+      ->notExists('field_scheduled_transition')
+      ->execute();
+
+    $sandbox['total'] = count($result);
+    $sandbox['current'] = 0;
+  }
+
+  $batch_size = 25;
+
+  $nids = \Drupal::entityQuery('node')
+    ->condition('status', 1)
+    ->condition('field_review_deadline', $formatted_three_weeks, '<=')
+    ->notExists('field_scheduled_transition')
+    ->range($sandbox['current'], $batch_size)
+    ->execute();
+
+  if (empty($nids)) {
+    $sandbox['#finished'] = 1;
+    return;
+  }
+
+  /** @var \Drupal\node\NodeInterface[] $nodes */
+  $nodes = \Drupal::entityTypeManager()
+    ->getStorage('node')
+    ->loadMultiple($nids);
+
+  foreach ($nodes as $node) {
+    $node->set('field_review_deadline', $formatted_one_year);
+    $node->save();
+    $sandbox['current']++;
+  }
+
+  \Drupal::messenger()->addStatus($sandbox['current'] . ' nodes with missing scheduled transition processed.');
+
+  if ($sandbox['current'] >= $sandbox['total']) {
+    $sandbox['#finished'] = 1;
+  } else {
+    $sandbox['#finished'] = ($sandbox['current'] / $sandbox['total']);
+  }
+}
+
+/**
+ * Update all published nodes missing a transition date with a review deadline
+ * more 3 weeks away.
+ */
+function epa_workflow_update_9004(&$sandbox) {
+  $now = \Drupal\Core\Datetime\DrupalDateTime::createFromTimestamp(time());
+  $three_weeks = $now->add(new DateInterval('P3W'));
+  $formatted_three_weeks = $three_weeks->format('Y-m-d\TH:i:s');
+
+  if (!isset($sandbox['total'])) {
+    // Query all published nodes that don't have a scheduled transition date, but
+    // do have a review deadline
+    $result = \Drupal::entityQuery('node')
+      ->condition('status', 1)
+      ->condition('field_review_deadline', $formatted_three_weeks, '>=')
+      ->notExists('field_scheduled_transition')
+      ->execute();
+
+    $sandbox['total'] = count($result);
+    $sandbox['current'] = 0;
+  }
+
+  $batch_size = 25;
+
+  $nids = \Drupal::entityQuery('node')
+    ->condition('status', 1)
+    ->condition('field_review_deadline', $formatted_three_weeks, '>=')
+    ->notExists('field_scheduled_transition')
+    ->range($sandbox['current'], $batch_size)
+    ->execute();
+
+  if (empty($nids)) {
+    $sandbox['#finished'] = 1;
+    return;
+  }
+
+  /** @var \Drupal\node\NodeInterface[] $nodes */
+  $nodes = \Drupal::entityTypeManager()
+    ->getStorage('node')
+    ->loadMultiple($nids);
+
+  foreach ($nodes as $node) {
+    // Nothing needs to change on the node, but triggering a save will create a
+    // scheduled transition date 3 weeks prior via the EPAPublished service.
```

Comment:
I assume you've validated this?  It's mildly surprising to me that it behaves this way, but is very convenient if true!